### PR TITLE
Fix pegsless inputs in beam and dosxyz GUI's

### PR DESCRIPTION
**Pull request by blakewalters**
Inputs now resemble those for egs_inprz and are similar to those
used to create pegs data in egs_gui.  In the process, fixed a bug
in dosxyz gui's get_val procedure.

### CODE REVIEW

File: HEN_HOUSE/omega/progs/gui/dosxyznrc/pegsless_egsnrc.tcl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 ###############################################################################
 #
-#  EGSnrc DOSXYZnrc graphical user interface: pegsless functions
+#  EGSnrc BEAMnrc graphical user interface: pegsless functions
```
Comment (rtownson) [inline HEN_HOUSE/omega/progs/gui/dosxyznrc/pegsless_egsnrc.tcl]: I think this change was an accident?

Comment (blakewalters) [inline HEN_HOUSE/omega/progs/gui/dosxyznrc/pegsless_egsnrc.tcl]: I'll take a look at this

Comment (blakewalters) [inline HEN_HOUSE/omega/progs/gui/dosxyznrc/pegsless_egsnrc.tcl]: Don't know if it was an accident or a bug.  I'll have to look in more detail...

**Comment by rtownson** [comment]
If you make changes push them to the PR #206 since the commit is already there, then we can close this PR. I don't know what happens if you pull the same commit from multiple PRs, maybe it doesn't matter?

**Comment by blakewalters** [comment]
See changes to PR #206.  There was an error in the element table used in the pegsless inputs!

File: HEN_HOUSE/omega/progs/gui/dosxyznrc/pegsless_egsnrc.tcl (hunk 1 above)
Comment (ojalaj) [inline HEN_HOUSE/omega/progs/gui/dosxyznrc/pegsless_egsnrc.tcl]: Is the problem described in #100 related to this?

**Comment by rtownson** [comment]
I'm closing this PR since this is now handled in PR #206.

**Pull request closed without merging**